include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -164,9 +166,39 @@ class BCX_API qrcode
             "The path to the configuration settings file."
         )
         (
+            "image,i",
+            value<bool>(&option_.image)->default_value(false)->zero_tokens(),
+            "Specifies that the QR code should be output in png format."
```

Comment:
The `Specifies` prefix for each help item isn't consistent with bx convention. Instead:
```
Write the QR code in png format.
```
Apply similar convention in other options.